test/probes/isUnsafeSpawn.spec.js

```diff
@@ -0,0 +1,81 @@
+// Import Node.js dependencies
+import { test } from "node:test";
+import assert from "node:assert";
+
+// Import Internal Dependencies
+import { parseScript, getSastAnalysis } from "../utils/index.js";
+import isUnsafeSpawn from "../../src/probes/isUnsafeSpawn.js";
+
+// CONSTANTS
+const kWarningUnsafeSpawn = "unsafe-spawn";
+
+test("should detect csrutil spawn command", () => {
+  const str = `
+    const { spawn } = require("child_process");
+    spawn("csrutil", ["status"]);
+  `;
+
+  const ast = parseScript(str);
+  const sastAnalysis = getSastAnalysis(str, isUnsafeSpawn)
+    .execute(ast.body);
+
+  const result = sastAnalysis.getWarning(kWarningUnsafeSpawn);
+  assert.equal(result.kind, kWarningUnsafeSpawn);
+  assert.equal(result.value, "csrutil");
+});
+
+test("should detect hidden csrutil spawn command", () => {
+  const str = `
+    const { spawn: hide } = require("child_process");
+    hide("csrutil", ["status"]);
+  `;
+
+  const ast = parseScript(str);
+  const sastAnalysis = getSastAnalysis(str, isUnsafeSpawn)
+    .execute(ast.body);
+
+  const result = sastAnalysis.getWarning(kWarningUnsafeSpawn);
+  assert.equal(result.kind, kWarningUnsafeSpawn);
+  assert.equal(result.value, "csrutil");
+});
```

Comment:
That one will fail on the CI. I'll take a deeper look at tracer to see how I can catch it.

if you have any pointer @fraxken feel free to share them